Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs

```diff
@@ -668,6 +667,15 @@ protected void ChangeEvent(int[] indexArray, SerializedProperty prop = null)
             }
         }
 
+        protected void DrawDrownDownProperty(Rect position, SerializedProperty prop, string[] options, GUIContent label)
```

Comment:
Not sure if this is what it's supposed to be, but was my best guess :)
```suggestion
        protected void DrawDropDownProperty(Rect position, SerializedProperty prop, string[] options, GUIContent label)
```